custom-vu/src/main/kotlin/jces1209/vu/page/AttachScreenShot.kt

```diff
@@ -0,0 +1,33 @@
+package jces1209.vu.page
+
+import jces1209.vu.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.OutputType
+import org.openqa.selenium.TakesScreenshot
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+abstract class AttachScreenShot(
+    protected val driver: WebDriver
+) {
+    protected var countBefore: Int = 0
+    abstract val screenShotLocator: By
+
+    fun makeScreenShot() {
+        (driver as TakesScreenshot).getScreenshotAs(OutputType.BYTES)
+    }
+
+    abstract fun pasteScreenShot(): Int
```

Comment:
it's not necessary to return Int after merging methods
```suggestion
    abstract fun pasteScreenShot()
```